### PR TITLE
Added Square Bouncing Animation

### PR DESCRIPTION
**Pull request by tvermaashutosh**
temp2

### CODE REVIEW

File: Art/Ash/style.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    animation-iteration-count: infinite;
+}
+
+@keyframes runMe{
+    /* from{
+        background-color: lightcoral;
+    }
+    to{
+        background-color: palegreen;
+    } */
```
Comment (sophiabrandt) [inline Art/Ash/style.css:17]: Please remove code that has no use (outcommented code).

File: include.js [hunk 2]
```diff
@@ -43,6 +43,13 @@ let cards = [
     author: 'Ahmed',
     githubLink: 'https://github.com/<ahmedalhamad7>'
   },
+  {
+    artName: 'Square Bounce', // change this to the name of your artwork
+    pageLink: './Art/Ash/triangle.html', // change this
+    imageLink: './Art/Ash/triangle.gif', // change this
+    author: 'Ash V', // use your name
+    githubLink: 'https://github.com/tvermaashutosh' // change this
+  },
```
Comment (sophiabrandt) [inline include.js:52]: Please remove the comments, there were only added in the README to explain what to do.

Also, please fill out the details with the name/folder path of _your contribution_.

Your folder name should be the Github username.  The file name should be the name of _your contribution_.

So, i.e. `pageLink: './Art/tvermaashutosh/index.html`.

File: Art/Ash/index.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (sophiabrandt) [inline Art/Ash/index.html:7]: Nitpick/Suggestion:

Change the `title` to something that's more suitable.


This is just a suggestion and not following it won't stop us from merging your contribution.

File: Art/Ash/index.html [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    <link rel="stylesheet" href="styling.css">
```
Comment (sophiabrandt) [inline Art/Ash/index.html:8]: The css file name you commited is `style.css`, so the `href` should point to that file.

**Review by sophiabrandt — CHANGES_REQUESTED** [review]
Thanks for this new contribution.

Please check the following checklist that you need to fulfill for us to be able to merge your request.

Please see the [Guide in README.md](https://github.com/zero-to-mastery/Animation-Nation#how-to-contribute) for further details.

##  Checklist
- [x] created a submission into the `./Art` folder
- [ ] the contribution folder in `./Art` is the contributor's Github name
- [ ] has a gif or png file of the animation
- [ ] can find the html 
- [ ] can find the style (css file)
- [ ] modified `include.js` **with the right details**

## Issues: 

- Please see the other suggestions in this PR.

Please refer to the `README.md` file for guidance and follow the guidelines.
If you have questions feel free to reach to the discord or ask them here.

**Comment by tvermaashutosh** [comment]
@sophiabrandt 
Please review and merge my PR. I have made all the suggested changes.

**Comment by sophiabrandt** [comment]
@tvermaashutosh Thank you for your contribution.

Unfortunately, there are still some items open:

Please see the [Guide in README.md](https://github.com/zero-to-mastery/Animation-Nation#how-to-contribute) for further details.

##  Checklist
- [x] created a submission into the `./Art` folder
- [ ] the contribution folder in `./Art` is the contributor's Github name
- [x] has a gif or png file of the animation
- [x] can find the html 
- [x] can find the style (css file)
- [ ] modified `include.js` **with the right details**


If you have questions feel free to reach to the discord or ask them here.

File: include.js (hunk 2 above)
Comment (sophiabrandt) [reply include.js:52]: The entry still does not meet the necessary requirements.

a) the folder names for your contribution should match your github name
b) the links should conform to the file names, i.e. `pageLink: './Art/tvermaashutosh/index.html`

**Review by sophiabrandt — CHANGES_REQUESTED** [review]

**Comment by tvermaashutosh** [comment]
@sophiabrandt 
Please review and merge my PR. I have made all the suggested changes again. 🙂🙂

**Pull request merged**